cupy_backends/cuda/api/runtime.pxd

```diff
@@ -613,9 +572,244 @@ cpdef enum:
     cudaReadModeNormalizedFloat = 1
 
 
+# This was a legacy mistake: the prefix "cuda" should have been removed
+# so that we can directly assign their C counterparts here. Now because
+# of backward compatibility and no flexible Cython macro (IF/ELSE), we
+# have to duplicate the enum. (CUDA and HIP use different values!)
```

Comment:
I am thinking whether we should rectify this mistake or not. For example, the attributes should be exposed under the same naming convention as other stuff in this module: `devAttr...`, with the prefix `cuda` removed and the following character being lowercase. Perhaps for backward compatibility we can make both the attributes (with old/new names) coexist, and in one or two releases we remove the old names?

Reply:
I think deprecating and removing is better

Reply:
@emcastillo Do you know how to emit warning if module-level attributes like these are accessed?

Reply:
The best solution I found is to utilize PEP 562, write a module-level `__getattr__()`, and emit a warning there. But it's Python 3.7+ only. Let's not worry about it now and do this in the next PR, as I have a second thought on the `cuda` prefix for runtime enums...😅